### PR TITLE
Add Feature/menu page sections

### PR DESCRIPTION
**Pull request by zihan0221**
Missing `MenuItem`

### CODE REVIEW

File: src/components/MenuSectionPage.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      })),
+    }))
+  );
+  }, []);
+  return (
+    <div className="min-h-screen flex flex-col items-center justify-center container mx-auto p-4">
+      {items.map((item, index) => (
+        <div key={index}>
+          <p className="text-lg font-medium my-4">{item.text}</p>
+          {item.merchants.map((merchant, i) => (
```
Comment (Yuankai619) [inline src/components/MenuSectionPage.jsx]: You missing the `key` attribute when use `map` to render list data.
this is very import to add `key`!!

**Comment by YuYutw123** [comment]
There are some problems in your commit.
* MenuSectionsPage.jsx should be under 'src/components/merchantPage' folder
* Please use Zustand to maintain your state

**Comment by YuYutw123** [comment]
Please fix the conflicts.

**Comment by YuYutw123** [comment]
Ok

**Review by YuYutw123 — APPROVED** [review]
Ok

**Pull request merged**